apis/traffic-access/traffic-access-WD.md

```diff
@@ -63,20 +64,21 @@ kind: TrafficTarget
 metadata:
   name: path-specific
   namespace: default
-destination:
-  kind: ServiceAccount
-  name: service-a
-  namespace: default
-  port: 8080
-specs:
-- kind: HTTPRouteGroup
-  name: the-routes
-  matches:
-  - metrics
-sources:
-- kind: ServiceAccount
-  name: prometheus
-  namespace: default
+spec:
+  destination:
+    kind: ServiceAccount
+    name: service-a
+    namespace: default
+    port: 8080
+  specs:
```

Comment:
Should we change the name of this to something else? `spec.specs` sounds ... weird.

Reply:
I propose `routes` instead of `specs`.

Reply:
The idea there, with the reference was for it to actually be pretty flexible, and more than just routes (especially for other protocols). Maybe `conditions`, `rules`?

Reply:
I went with `rules` as the specification doc already uses this terminology.